data/FilterList.json

```diff
@@ -16590,4 +16591,15 @@
     "updatedDate": "2019-01-15T03:00:00",
     "viewUrl": "https://raw.githubusercontent.com/cbuijs/shallalist/master/webtv/urls"
   }
```

Comment:
you missed the comma and brace on line 16594

Reply:
I'm siding with Bogachenko on this one. A paragraph should normally be opened with `{` , and should be closed with `},` with a comma.

( `}` without a comma if it's the last paragraph).